src/status_im/ui/screens/wallet/choose_recipient/events.cljs

```diff
@@ -11,7 +11,10 @@
       (assoc-in db [:wallet :send-transaction :camera-flashlight] toggled-state))))
 
 (defn choose-address-and-name [db address name amount]
-  (update-in db [:wallet :send-transaction] assoc :to-address address :to-name name :amount amount))
+  (cond-> db
+          true (update-in [:wallet :send-transaction] assoc :to-address address :to-name name)
```

Comment:
just do `(cond-> (update-in db [:wallet :send-transaction] assoc :to-address address :to-name name)`

Reply:
Good point!